src/linear_algebra/span.lean

```diff
@@ -557,6 +557,17 @@ begin
   exact ⟨y, ⟨hyd, by simpa only [span_le, singleton_subset_iff]⟩⟩,
 end
 
+/-- The span of a finite set is compact in the lattice of submodules. -/
+lemma finite_span_is_compact_element (S : finset M) :
```

Comment:
Can you add a variant of this lemma using `set.finite`?

Reply:
OK I added it.